jenkins-x-lint.yml

```diff
@@ -20,11 +20,11 @@ pipelineConfig:
               containerOptions:
                 resources:
                   limits:
+                    cpu: 4
+                    memory: 8Gi
```

Comment:
@abayer do you have any issue with increasing the limit here?

Reply:
I do not, no.

Reply:
:-)